core/src/test/java/io/questdb/log/MockAlertTarget.java

```diff
@@ -58,15 +64,21 @@ public void run() {
                 serverSkt = new ServerSocket(portNumber);
                 serverSkt.setReuseAddress(true);
                 serverSkt.setSoTimeout(5000);
+                if (portNumber == 0) {
+                    portNumber = serverSkt.getLocalPort();
+                }
+
                 onTargetStart.run();
                 clientSkt = serverSkt.accept();
+
                 in = new BufferedReader(new InputStreamReader(clientSkt.getInputStream()));
                 out = new PrintWriter(clientSkt.getOutputStream(), true);
+
```

Comment:
Reuse address combined with no linger created a 'storm' of packet retransmissions on windows that sometimes failed the test .